rosa/terraform/network.tf

```diff
@@ -0,0 +1,244 @@
+# SPDX-FileCopyrightText: 2024 SAP edge team
```

Comment:
When running terraform apply multiple times on this ROSA HCP deployment, Terraform detects
  "drift" because ROSA/OpenShift automatically adds tags to subnets (like
  kubernetes.io/cluster/*, api.openshift.com/*, etc.) that aren't managed in the Terraform
  configuration.

Could you pls fix this?